instances/technique/biocytinStaining.jsonld

```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/biocytinStaining",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
-  "description": null,
+  "definition": "Biocytin staining is a technique commonly used in electrophysiology for post-hoc recovery of morphological details of nerve cells.",
+  "description":  "Biocytin staining is a technique commonly used in electrophysiology for post-hoc recovery of morphological details. Biocytin is a histological stain for nerve cells that is included in the recording electrode. It allows for the visualisation of the dendritic arborization and the regions targeted by the axons of the recorded neurons.",
```

Comment:
I would typically not reuse the definition again in the description to avoid redundancy for the reader. Maybe replace definition with:
"In 'biocytin staining' the chemical compound biocytin is used to highlight morphological details of nerve cells." 

I would rephrase the description to:
"Biocytin staining is a technique commonly used in combination with intracellular electrophysiology for post-hoc recovery of morphological details of the recorded neurons. For this, the chemical compound biocytin is included in the recording electrode in order to fill the recorded cell. It allows for the visualisation of the dendritic arborization and the regions targeted by the axons of the recorded neurons."

I would add the synoyms "biocytin filling" and "biocytin labeling"